pyo3-ffi/src/lib.rs

```diff
@@ -0,0 +1,206 @@
+//! Raw FFI declarations for Python's C API.
+//!
+//! This module provides low level bindings to the Python interpreter.
```

Comment:
It's not a module anymore 😊 (same for other uses of "module")
```suggestion
//! This crate provides low level bindings to the Python interpreter.
```